### PR TITLE
feat: switch badge logic for editor

### PR DESCRIPTION
**Pull request by alexnm**
Part 2 of the subscription badge update
* Pro badge removed
* Free badge updated to use the new component/design
* Personal space renamed to `Personal`
* Tooltip kept, but adapted with content from sidebar
* Cleanup

Screenshot 2022-10-21 at 18 18 48

Screenshot 2022-10-21 at 18 22 55

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/7048/commits/ed8d934f1cd3748e7574120f5a0bb98257cd6006 is at https://pr7048.build.csb.dev/s/new.

File: packages/app/src/app/components/WorkspaceSelect/elements.ts [hunk 1]
```diff
@@ -1,38 +0,0 @@
...
-  top: 1px; // 👌
-`;
-
-const MenuItem = styled(Stack)`
-  height: 10;
-  text-align: left;
-  background-color: ${({ theme }) => theme.colors.grays[600]};
-  border-bottom: 1px solid ${({ theme }) => theme.colors.grays[500]};
-
-  &[data-reach-menu-item][data-component='MenuItem'][data-selected] {
```
Comment (tristandubbeld) [inline packages/app/src/app/components/WorkspaceSelect/elements.ts:27]: Do we not need these `selected` styles anymore?

File: packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx [hunk 2]
```diff
@@ -42,7 +34,16 @@ export const Header = () => {
     >
       <Stack align="center">
         <AppMenu />
-        {renderWorkspace()}
+        {activeTeamInfo && (
+          <WorkspaceName
+            name={
+              activeTeamInfo.id === personalWorkspaceId
+                ? 'Personal'
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx:41]: 🙌

**Review by tristandubbeld — APPROVED** [review]

**Review by zehfernandes — APPROVED** [review]

File: packages/app/src/app/components/WorkspaceSelect/elements.ts (hunk 1 above)
Comment (alexnm) [reply packages/app/src/app/components/WorkspaceSelect/elements.ts:27]: they were't used before either, there's no selected state in the workspace selector

**Pull request merged**